Autocomplete/predefined-items-and-recent-searches/src/recent-searches.js

```diff
@@ -0,0 +1,29 @@
+import { createRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+const orgRecentSearches = createRecentSearchesPlugin({
+  key: 'RECENT_SEARCH',
+  limit: 5,
+});
+
+export const recentSearches = {
```

Comment:
Currently `@algolia/autocomplete-plugin-recent-searches` doesn't allow custom template manipulation. That's why I'm spreading and injecting the header template.
Should we introduce an API in autocomplete to allow altering templates for plugins?

Reply:
Yes—we need to find a proper solution for this.